### PR TITLE
arvind2021reproducibility

### PR DESCRIPTION
**Pull request by MustansirM**
This is the first version of our camera-ready paper.

We had one issue with LaTeX formatting. We have used \DeclareMathOperator to define a few activation functions. However, in the final PDF, these operators are not formatted differently and are indistinguishable from the rest of the text. The compilation logs contain several warnings, but we were able to obtain the camera ready PDF successfully except for the aforementioned issue.

### CODE REVIEW

File: arvind2021reproducibility/journal/metadata.yaml [hunk 1]
```diff
@@ -5,59 +5,45 @@
...
 # You can get a DOI for your code from Zenodo, or an SWH identifier from
 # Software Heritage.
 #   see https://guides.github.com/activities/citable-code/
 code:
-  - url: https://github.com/rescience-c/template
-  - doi: 
-  - swh: 
+  - url: https://github.com/mayurak47/Reproducibility_Challenge
+  - doi: 10.5281/zenodo.4696245
+  - swh:
```
Comment (koustuvsinha) [inline arvind2021reproducibility/journal/metadata.yaml:46]: Please add the Software Heritage Foundation link of your code (instructions in README)

**Comment by koustuvsinha** [comment]
Regarding the issue in declaring a math operator, you could probably try declaring it with `\texttt` to be distinguishable from normal text:

```
\DeclareMathOperator{\snake}{\texttt{snake}}
```

which results in this:

Screen Shot 2021-04-18 at 5 00 16 PM

**Comment by MustansirM** [comment]
The Software Heritage Foundation link has been added and we've incorporated your suggestion for changing the format of Math Operators. Thank you for the assistance.

**Comment by koustuvsinha** [comment]
LGTM! @sashavor can you review the content? Thanks!

**Comment by sashavor** [comment]
The article pdf looks good!

**Pull request merged**